reference/configuration/framework.rst

```diff
@@ -196,9 +197,35 @@ see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
 form
 ~~~~
 
+.. _form-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``false``
+
+Whether or not to enable support for the Form component.
```

Comment:
I guess this description matches the succinct style of the rest of the configuration reference, but I'd like to provide more information about the consequences of settings this option. Examples: If I don't use the Form component and I disable it, will my application performance improve? If I disable the Form component, will some Symfony related feature stop working?